chore: Deprecate kyma-cli v2 (#2095)
kyma-project/cli: README.md

# Kyma CLI

> > [!WARNING]

Author: It's based on [this](https://github.com/orgs/community/discussions/16925) article

> > All current commands are deprecated and new `v3` commands will be developed within `alpha` command group.

Reviewer: ```suggestion
> Version `v2` of kyma CLI is deprecated. This implies that all commands available within version `v2` are also deprecated. We started designing `v3` commands that will be first released within `alpha` command group.
```

> > Version `v2` of kyma CLI is deprecated. This implies that all commands available within version `v2` are also deprecated. We started designing `v3` commands that will be first released within `alpha` command group.

Reviewer 2: ```suggestion
> The Kyma CLI version `v2`, with all commands available within this version, is deprecated. We've started designing the `v3` commands that will be first released within the `alpha` command group.
```
